src/platform/mod.rs

```diff
@@ -15,47 +15,20 @@
 //!
 //! However only the module corresponding to the platform you're compiling to will be available.
 
-#[cfg(target_os = "android")]
+#[cfg(android)]
```

Comment:
I wonder why it fails here? If we can't use android like that I'd suggest to change so the failure gets resolved.

Reply:
the solution @maroider came up with was to change the aliases to have `os_*` prefix

Reply:
Could have platform_android, so it'll look like x11/wayland.